src/lib/default-template-processor.ts

```diff
@@ -34,9 +34,9 @@ export class DefaultTemplateProcessor implements TemplateProcessor {
    * @param strings The string literals. There are always at least two strings,
    *   event for fully-controlled bindings with a single expression.
    */
-  handleAttributeExpressions(
+  public handleAttributeExpressions(
```

Comment:
Explicit `public` strikes me as noisy. `public` being the default seems reasonable to me.